addons/token-exchange/token_exchanger_addon.go

```diff
@@ -76,11 +76,13 @@ func (a *TokenExchangeAddon) Manifests(cluster *clusterv1.ManagedCluster, addon
 		ClusterName           string
 		AddonInstallNamespace string
 		Image                 string
+		OwnerMirrorPeer       string
```

Comment:
Why do we need this? Addon is not owned by any specific MirrorPeer.

Reply:
I've removed this ... Added it earlier before the controller. We don't need it anymore but it works well =)